packages/ember-data/lib/adapters/rest-adapter.js

```diff
@@ -838,3 +838,13 @@ function endsWith(string, suffix) {
     return string.endsWith(suffix);
   }
 }
+
+Ember.defineProperty(RESTAdapter, 'maxUrlLength', {
```

Comment:
Sorry, I missed that. What could we do for IE8 ? I can at least put  this defineProperty inside a  Ember.platform.hasPropertyAccessors block, but how could I do it for IE8 ? (Am I missing something or Ember.deprecateProperty is just a noop on IE8 ?)